include/mbedtls/ssl_internal.h

```diff
@@ -1064,7 +1064,7 @@ int mbedtls_ssl_decrypt_buf( mbedtls_ssl_context const *ssl,
                              mbedtls_record *rec );
 
```

Comment:
> Make SSL fn's shared betw'n ssl_tls.c and ssl_msg.c public-internal

Pls dn abbrvt lk ths

“Declare newly shared ssl functions in a header file” would be sufficiently descriptive and is under the (soft) 70 char limit.

A'o g's f'r “Introduce separate source files for TLS 1.2 HS rd/wr functions”

Reply:
Was that necessary? Anyway, I'm ok with your suggestion.